test/integration/test_ibis.py

```diff
@@ -7,7 +7,7 @@
 import torchvision
 
 from superduperdb.backends.filesystem.artifacts import FileSystemArtifactStore
-from superduperdb.backends.ibis.data_backend import IbisDataBackend
+from superduperdb.backends.ibis.datastore import IbisDataBackend
```

Comment:
Data-backend was chosen to make the abstraction even more inclusive than Datastore.